models/engine/db_storage.py

```diff
@@ -74,3 +74,18 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """returns the object based on the class and its ID, or None if not found"""
```

Comment:
Please double-check the pycodestyle, it has 5 characters more than allowed.

Reply:
ok it was solved